test/test_app_details.py

```diff
@@ -86,30 +89,26 @@ def test_get_store_path_with_relative_path(env_app_root):
 
 # ==== get_app_base_url ====
 
-@patch(GET_MANIFEST_JSON, return_value=manifest_path('pre_install.json'))
-def test_get_app_base_url_returns_empty_string_when_app_id_missing_from_manifest(mock_manifest):
+def test_get_app_base_url_returns_empty_string_when_app_id_missing_from_env():
     assert qpylib.get_app_base_url() == ''
 
-@patch(GET_MANIFEST_JSON, return_value=manifest_path('installed_no_console_ip.json'))
-def test_get_app_base_url_returns_empty_string_when_host_cannot_be_determined(mock_manifest):
+def test_get_app_base_url_returns_empty_string_when_host_cannot_be_determined():
```

Comment:
This test is exactly the same as the one above. Think you need to supply the env_qradar_app_id fixture.